ynr/apps/bulk_adding/forms.py

```diff
@@ -240,7 +244,7 @@ def clean(self):
 
 
 class NameOnlyPersonForm(forms.Form):
-    name = forms.CharField(
+    name = StrippedCharField(
```

Comment:
Just checking, does this require a migration?

Reply:
I've checked again, it does not.